src/official-account/official-account.ts

```diff
@@ -252,6 +254,40 @@ class OfficialAccount extends EventEmitter {
     return ret.body
   }
 
+  async uploadTempFile (args: {
```

Comment:
I'd like to suggest renaming `uploadTempFile` to a reasonable name, like `sendFile()`?

Because I guess our OA puppet will always upload and send a file. Please correct me if I'm not right.

Reply:
Yes, we should keep the same naming style with the wechaty. Here is my consideration:
- there are `temp file` and `permanent file` in tencent official api
- we can set the `isTempFile` symbol to mark if this file is uploaded as temp file or permanent file 

@huan  how do you think about ?

Reply:
What's your purposal?

Reply:
`sendFile` can send `temp file` and `permanent file` to tencent official api, which has the different api url. So, we need a param to declare which api it touches.